src/HieDb/Utils.hs

```diff
@@ -240,3 +242,39 @@ identifierTree nd@HieTypes.Node{ nodeChildren } =
     { rootLabel = nd { nodeChildren = mempty }
     , subForest = map identifierTree nodeChildren
     }
+
+generateExports :: FilePath -> [AvailInfo] -> [ExportRow]
+generateExports fp = concatMap generateExport where
+  generateExport :: AvailInfo -> [ExportRow]
+  generateExport (Avail n)
+    = [ExportRow
+        { exportHieFile = fp
+        , exportName = nameOccName n
+        , exportMod = moduleName $ nameModule n
+        , exportUnit = moduleUnit $ nameModule n
+        , exportParent = Nothing
+        , exportParentMod = Nothing
+        , exportParentUnit = Nothing
+        , exportIsDatacon = False
+        }]
+  generateExport (AvailTC name pieces fields)
+    = ExportRow
+        { exportHieFile = fp
+        , exportName = nameOccName name
+        , exportMod = moduleName $ nameModule name
+        , exportUnit = moduleUnit $ nameModule name
+        , exportParent = Nothing
+        , exportParentMod = Nothing
+        , exportParentUnit = Nothing
+        , exportIsDatacon = False}
+    : [ExportRow
+        { exportHieFile = fp
+        , exportName = occ
+        , exportMod = moduleName $ nameModule name
```

Comment:
These needs to be the module and unit for the child names, not the parent name

Reply:
Perhaps you can add a `data family` with the family defined in one module and the instance defined in a parent module to test this case.

Reply:
Ahh, thanks! Fixed now